### PR TITLE
DESC and README

### PR DESCRIPTION
**Pull request by junlueZH**
part of https://github.com/insightsengineering/teal/issues/668

Also closes https://github.com/insightsengineering/teal/issues/686

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -55,6 +55,15 @@ Suggests:
...
     knitr
+Remotes:
+    insightsengineering/teal.code@*release
+    insightsengineering/teal.reporter@*release
+    insightsengineering/teal.slice@*release
+    insightsengineering/teal.data@*release
+    insightsengineering/teal.logger@*release
+    insightsengineering/teal.widgets@*release
+    insightsengineering/scda@*release
+    insightsengineering/scda.2021@*release
```
Comment (nikolas-burkoff) [inline DESCRIPTION:66]:
```suggestion
Remotes:
    insightsengineering/teal.code@*release,
    insightsengineering/teal.reporter@*release,
    insightsengineering/teal.slice@*release,
    insightsengineering/teal.data@*release,
    insightsengineering/teal.logger@*release,
    insightsengineering/teal.widgets@*release,
    insightsengineering/teal.transform@*release,
    insightsengineering/scda@*release,
    insightsengineering/scda.2021@*release
```

**Review by nikolas-burkoff — APPROVED** [review]
👍

File: README.md [hunk 2]
```diff
@@ -33,16 +33,18 @@ See these packages for more information about how to use the different parts of
...
 Sys.setenv(GITHUB_PAT = "your_access_token_here")
-if (!require("devtools")) install.packages("devtools")
-devtools::install_github("insightsengineering/teal@*release")
+if (!require("remotes")) install.packages("remotes")
+remotes::install_github("insightsengineering/teal@*release")
 ```
 
 In order to run many of the examples you will also need to install the [`scda`](https://insightsengineering.github.io/scda) package.
 
+A stable release of all `NEST` packages from June 2022 is also available [here](https://github.com/insightsengineering/depository#readme).
```
Comment (nikolas-burkoff) [inline README.md:46]:
```suggestion
A stable release of all `NEST` packages from June 2022 is available [here](https://github.com/insightsengineering/depository#readme).
```

File: tests/testthat/test-get_rcode.R [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
 
+  r_code_from_chunks <- get_rcode(chunks = ch)
+  r_code_from_header <- sprintf("\n\n%s\n", paste(get_rcode_header(), collapse = "\n"))
+
+  # removing the Date line from the header as the seconds may be different
+  # in the two strings
   testthat::expect_identical(
-    get_rcode(chunks = ch),
-    sprintf("\n\n%s\n", paste(get_rcode_header(), collapse = "\n"))
+    strsplit(r_code_from_chunks, "\n")[[1]][c(1:5, 7:14)],
```
Comment (Polkas) [inline tests/testthat/test-get_rcode.R:10]: 🤡 if works then it works`:D

**Pull request merged**